pkg/controllermanager/controller/shoot/shoot_maintenance_control.go

```diff
@@ -177,6 +177,7 @@ func (c *defaultMaintenanceControl) Maintain(shootObj *gardencorev1beta1.Shoot,
 		if updatedKubernetesVersion != nil {
 			s.Spec.Kubernetes.Version = *updatedKubernetesVersion
 		}
+		s.Spec.Provider.EnableContainerD = true
```

Comment:
Same like above

Reply:
Ok, i will remove it but do you want to keep it in the GEP?

Reply:
Yes, I think it makes sense to remove it from the GEP

Reply:
Ok